apps/docs/src/guide/errors/error-codes.md

```diff
@@ -0,0 +1,71 @@
+# Error Codes
+
+Here is a list of the expected error codes the SDK can throw. These error codes are used to help understand the error that has been thrown with potential remedies.
+
+Error Codes | Description
+--- | ---
+ABI_MAIN_METHOD_MISSING | When your ABI does not have a main method. This can be resolved by adding a main method to your ABI.
+ABI_TYPES_AND_VALUES_MISMATCH | When the arguments supplied to the function do not match the minimum required input length. Check that the arguments supplied to the function, match the required type.
+ACCOUNT_REQUIRED | When the account has not been set for the contract.
+CONVERTING_FAILED | When the `bn` function failed to convert a value supplied.
```

Comment:
```suggestion
CONVERTING_FAILED | When the `bn` function failed to convert a value supplied. Check it's valid input types.
```

Reply:
CONVERTING_FAILED
When a converting a big number into an incompatible format.
Ensure that the value you've supplied to the big number, is compatibility with the value you are converting to.

Reply:
> compatibility

compatible

But otherwise agreed 👍🏻

Reply:
> compatible

[511923f](https://github.com/FuelLabs/fuels-ts/pull/1866/commits/511923fe320bbb9c3f691d28bed95c59d8f418b6)